stdlib/public/Reflection/TypeLowering.cpp

```diff
@@ -36,75 +34,74 @@ namespace swift {
 namespace reflection {
 
 void TypeInfo::dump() const {
-  dump(std::cerr);
+  dump(stderr);
 }
 
 namespace {
 
 class PrintTypeInfo {
-  std::ostream &OS;
+  FILE *file;
   unsigned Indent;
 
-  std::ostream &indent(unsigned Amount) {
+  FILE * &indent(unsigned Amount) {
     for (unsigned i = 0; i < Amount; ++i)
-      OS << ' ';
-    return OS;
+      fprintf(file, " ");
+    return file;
   }
 
-  std::ostream &printHeader(const std::string &name) {
-    indent(Indent) << '(' << name;
-    return OS;
+  FILE * &printHeader(const std::string &name) {
+    fprintf(indent(Indent), "(%s", name.c_str());
+    return file;
   }
 
-  template<typename T>
-  std::ostream &printField(const std::string &name, const T &value) {
+  FILE * &printField(const std::string &name, const std::string &value) {
     if (!name.empty())
-      OS << " " << name << "=" << value;
+      fprintf(file, " %s=%s", name.c_str(), value.c_str());
     else
-      OS << " " << value;
-    return OS;
+      fprintf(file, " %s", value.c_str());
+    return file;
   }
 
   void printRec(const TypeInfo &TI) {
-    OS << "\n";
+    fprintf(file, "\n");
 
     Indent += 2;
     print(TI);
     Indent -= 2;
   }
 
   void printBasic(const TypeInfo &TI) {
-    printField("size", TI.getSize());
-    printField("alignment", TI.getAlignment());
-    printField("stride", TI.getStride());
-    printField("num_extra_inhabitants", TI.getNumExtraInhabitants());
-    printField("bitwise_takable", TI.isBitwiseTakable());
+    printField("size", std::to_string(TI.getSize()));
+    printField("alignment", std::to_string(TI.getAlignment()));
+    printField("stride", std::to_string(TI.getStride()));
+    printField("num_extra_inhabitants", std::to_string(TI.getNumExtraInhabitants()));
+    printField("bitwise_takable", TI.isBitwiseTakable() ? "1" : "0");
```

Comment:
Might as well go with "true" and "false" if you're changing it anyway.

Reply:
Done